### DISCUSSION TITLE
Lint error (input-otp)

### DISCUSSION THREAD

**Discussion by tomyroydotcom** [shadcn-ui/ui#3367] [Q&A] [answered]
Hi,

Im pretty new to coding and recently added all the shadcn components to my project. It was linting fine before adding the input-otp

here's the error
![image](https://github.com/shadcn-ui/ui/assets/96185908/908590f3-ea70-4c75-8c47-ca5ad652c4d0)

Being a total noob I have no idea how to correct this. Where's the SlotProps type located ? Is that normal ? Is my "lint" ok ?

**Comment by tomyroydotcom** [comment, marked as answer]
So it threw the error because it COULD'VE been undefined.

Fixing it by adding an error handling removes the error.
![image](https://github.com/shadcn-ui/ui/assets/96185908/ddee9a2e-a719-47ed-8389-154591dec88f)
instead of 
![image](https://github.com/shadcn-ui/ui/assets/96185908/bda1fe94-0417-4f79-b7fb-54639b26b86a)

This is probably something really minimal and not needed for most I guess

**Reply by pricetula** [reply]
Build failure is caused by this issue
